fire/core.py

```diff
@@ -161,6 +161,20 @@ class FireError(Exception):
   """
```

Comment:
Should FireError be private?
They are stored in the FireTraces, which are now accessible outside of Fire via the FireExit.